src/components/common/AppMenu.vue

```diff
@@ -24,7 +24,10 @@
         to="/portfolio"
         exact="exact"
         title="Portfolio"
-        @click.native="close"
+        @click.native="
+          close()
+          scrollToTop()
```

Comment:
I think you are missing a ;

Reply:
Note: instead of calling both functions here you could also create a "handleMenuClick" function who executes both functions to avoid the duplications.
Or just add the logic to the close function